storybook/storybook-react/src/IconButton/IconButton.stories.tsx

```diff
@@ -12,11 +12,12 @@ const meta = {
   component: IconButton,
   args: {
     label: 'Sluiten',
+    onBackground: undefined,
+    disabled: false,
+    size: undefined,
```

Comment:
This doesn't do anything, I think. Maybe we should make `level-5` undefined, similar to what we do with `onBackground`?